packages/core/src/utils/signature.ts

```diff
@@ -0,0 +1,8 @@
+import { createHmac } from 'node:crypto';
+
+export const generateSignature = (signingKey: string, payload: Record<string, unknown>) => {
+  const hmac = createHmac('sha256', signingKey);
+  const payloadString = JSON.stringify(payload);
+  hmac.update(payloadString);
+  return `sha256=${hmac.digest('hex')}`;
+};
```

Comment:
does it have an async version? since the payload may be large

Reply:
Good call, updated.